format/FlightSql.proto

```diff
@@ -867,6 +867,69 @@ enum SqlSupportsConvert {
   SQL_CONVERT_VARCHAR = 19;
 }
 
+/*
+ * Represents a request to retrieve information about data type supported ona Flight SQL enabled backend.
+ * Used in the command member of FlightDescriptor for the following RPC calls:
+ *  - GetSchema: return the schema of the query.
+ *  - GetFlightInfo: execute the catalog metadata request.
+ *
+ * The returned schema will be:
+ * <
+ *   type_name: utf8 not null (The name of the data type, for example: VARCHAR, INTEGER, etc),
+ *   data_type: int not null (The SQL data type),
+ *   column_size: int (The maximum column size that the server supports for this data type.
+ *                     For numeric data, this is the maximum precision.
+ *                     For string data, this is the length in characters.
+ *                     For datetime data types, this is the length in characters of the string representation.
+ *                     NULL is returned for data types where column size is not applicable.),
+ *   literal_prefix: utf8 (Character or characters used to prefix a literal, NULL is returned for
+ *                         data types where a literal prefix is not applicable.),
+ *   literal_suffix: utf8 (Character or characters used to terminate a literal,
+ *                         NULL is returned for data types where a literal suffix is not applicable.),
+ *   create_params: utf8 (A list of keywords, separated by commas, corresponding
```

Comment:
Would this make more sense as List[String not null]?

Reply:
Also now that I look at it - we should be careful not to 1:1 copy language from the ODBC specification.

Reply:
Changed to a List 

Fixed